lib/va_profile/contact_information/v2/service.rb

```diff
@@ -214,7 +214,12 @@ def get_person_transaction_status(transaction_id)
         private
 
         def verify_vet360_id!
-          raise 'ContactInformationV2 - Missing User VAProfile_ID' if @user&.vet360_id.blank?
+          return if @user&.vet360_id.present?
+
+          raise Common::Exceptions::UnprocessableEntity.new(
+            detail: 'User does not have a VA Profile ID',
+            source: 'VAProfile::ContactInformation::V2::Service'
```

Comment:
Not a blocking comment, but should we use `self.class.name` here instead of `'VAProfile::ContactInformation::V2::Service'` (avoid potential future mismatch between class name and the exception source)?